.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+on:
+  push:
+    tags: ["v*"]
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-20.04
+    env:
+      SBT_NATIVE_CLIENT: true
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+          cache: sbt
+      - run: sbt versionCheck
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+      - run: sbt ci-release
```

Comment:
It seems the `env` applies to the `versionCheck` step, not the `ci-release` one.

Reply:
`Play` release is broken too then.

Reply:
Thank you, I edited the file there too!